src/main/resources/data-repository-openapi.yaml

```diff
@@ -980,6 +980,24 @@ paths:
           schema:
             type: object
 
+
+  '/api/repository/v1/configuration':
+    get:  ## retrieve
+      description: Retrieve the repository configuration information
+      operationId: retrieveRepositoryConfig
+      tags:
+      - repository
```

Comment:
Can you make this `- unauthenticated`? We want this to be similar to the /status endpoint.